### DISCUSSION TITLE
Tracker File Name Modification

### DISCUSSION THREAD

**Discussion by ElementAngles** [immersivecognition/unity-experiment-framework#153] [General]
Can the file name of the file generated by a tracker be modified to include information from the initial UI inputs (such as Session) and/or particular settings in the JSON file (e.g., task_prioritization: "Group_A")?

**Comment by jackbrookes** [comment]
Hi! No, I don't see why you would use this. The best way would be to add `task_prioritization` to `settings to log` in the session. That way you will get e.g. `Group_A` in each row. Then you can use the `filepath` reference column to load in the tracker data. I can explain more if it isn't clear!

**Reply by ElementAngles** [reply]
Thanks for the reply Jack! I've probably abused your program a bit. 

A little bit of context. My task in Unity is a pursuit tracking task that lasts about a minute. I've written a tracking script that adds several additional columns of data into the tracking file. This data is collected from UXF UI inputs, JSON settings, and other changes that occur during the trial. The data reflects the conditions of both the pursuit tracking task and a secondary task that is sometimes paired with the pursuit tracking task. The secondary task is ran through a program separate from Unity. On some trials, only the secondary task is performed. For a variety of reasons, I'm not running multiple trials per se within the unity program. Instead an actual trial number input and the secondary task condition info is recorded on the initial UXF UI window for each run.

Anyway, my tracker file contains everything that I need for analysis and the program runs as expected. I'd simply like to give the files different names (e.g., ppid, and then the actual trial number pulled from the UI input [or I could use the Session input since it is not being used by me]) so I can put them in the same folder for post processing. My post processing script collapses the data from all of the trials into a single file that contains the needed statistics for each trial within a single row (e.g., mean error, RMS, etc.) for analysis.  

So, if there was a place to modify the C# script within UXF -- to add in the ppid and session input to the tracker file name, that's one possibility that could  work for my purposes.

Thanks in advance.
